bukkit/build.gradle

```diff
@@ -1,5 +1,5 @@
 dependencies {
-    compileOnly group: "org.spigotmc", name: "spigot-api", version: "1.8.8-R0.1-SNAPSHOT"
+    compileOnly group: "org.spigotmc", name: "spigot-api", version: "1.16.5-R0.1-SNAPSHOT"
```

Comment:
No - we maintain compatibility back through 1.8.8. Why would you bump this?

Reply:
A normal update because time is moving on and everyone hates outdated Minecraft versions.

Reply:
No, this just introduces the risk that someone might depend on a newer API version and break older versions. Revert this.